x-pack/plugins/cases/public/components/all_cases/utils.ts

```diff
@@ -28,3 +34,52 @@ export const parseUrlQueryParams = (parsedUrlParams: ParsedUrlQueryParams): Part
 
   return urlParams;
 };
+
+export const mergeSelectedColumnsWithConfiguration = ({
+  selectedColumns,
+  casesColumnsConfig,
+}: {
+  selectedColumns: CasesColumnSelection[];
+  casesColumnsConfig: CasesColumnsConfiguration;
+}): CasesColumnSelection[] => {
+  // selectedColumns is the master
```

Comment:
you could get rid of this comment, the code is self explanatory enough to make this unnecessary